discovery-provider/integration_tests/challenges/test_track_upload_challenge.py

```diff
@@ -135,6 +150,17 @@ def test_track_upload_challenge(app):
         assert user_challenge.current_step_count == 1
         assert not user_challenge.is_complete
 
+        # Ensure unlisted track is not counted
+        session.add(unlisted_track)
+        bus.dispatch(ChallengeEvent.track_upload, 30000000, 1)
```

Comment:
should this be 30000001?

Reply:
shouldn't affect the test but for consistency yep, will change